### ISSUE TITLE
Why not to reduce learning rate during training?

### ISSUE THREAD

**Issue by wikiwen** [NVlabs/stylegan2-ada-pytorch#201]
I found the code use a constant learning rate. Will it be better to reduce learning rate during training? For example, fid vary 400 -> 50 ->200 if use constant lr. If lr is reduced when fid=50, maybe fid will be lower.

**Comment by wikiwen** [comment]
I have complete this experiment, But found it's useless.
